core-plugins/src/e2e-test/features/errorcollector/ErrorCollector.feature

```diff
@@ -0,0 +1,74 @@
+    When Select plugin: "JavaScript" from the plugins list as: "Transform"
+    When Expand Plugin group in the LHS plugins list: "Error Handlers and Alerts"
+    When Select plugin: "Error Collector" from the plugins list as: "Error Handlers and Alerts"
+    Then Move plugins: "ErrorCollector" by xOffset 200 and yOffset 200
+    When Expand Plugin group in the LHS plugins list: "Sink"
+    When Select plugin: "File" from the plugins list as: "Sink"
+    Then Connect plugins: "File" and "JavaScript" to establish connection
+    Then Connect node type: "error" of plugin: "JavaScript" to plugin: "ErrorCollector"
+    Then Connect plugins: "ErrorCollector" and "File2" to establish connection
+    Then Navigate to the properties page of plugin: "File"
+    Then Enter input plugin property: "referenceName" with value: "firstName"
+    Then Enter input plugin property: "path" with value: "gcsSourceBucket1"
+    Then Select dropdown plugin property: "format" with option value: "csv"
+    Then Click plugin property: "skipHeader"
+    Then Click plugin property: "enableQuotedValues"
+    Then Click on the Get Schema button
+    Then Verify the Output Schema matches the Expected Schema: "firstNameOutputSchema"
+    Then Validate "File" plugin properties
+    Then Close the Plugin Properties page
+    Then Navigate to the properties page of plugin: "JavaScript"
+    Then Replace textarea plugin property: "script" with value: "transformScript"
+    Then Validate "JavaScript" plugin properties
+    Then Verify the Output Schema matches the Expected Schema: "firstNameOutputSchema"
+    Then Validate "JavaScript" plugin properties
+    Then Close the Plugin Properties page
+    Then Navigate to the properties page of plugin: "ErrorCollector"
+    Then Click on the Get Schema button
+    Then Verify the Output Schema matches the Expected Schema: "errorCollectorOutputSchema"
+    Then Validate "ErrorCollector" plugin properties
+    Then Close the Plugin Properties page
+    Then Navigate to the properties page of plugin: "File2"
+    Then Enter input plugin property: "referenceName" with value: "errorSink"
+    Then Enter input plugin property: "path" with value: "gcsTargetBucket"
+    Then Select dropdown plugin property: "format" with option value: "csv"
+    Then Validate "File2" plugin properties
+    Then Close the Plugin Properties page
+    Then Save the pipeline
+    Then Preview and run the pipeline
+    Then Wait till pipeline preview is in running state
+    Then Open and capture pipeline preview logs
+    Then Verify the preview run status of pipeline in the logs is "succeeded"
+    Then Close the pipeline logs
+    Then Close the preview
+    Then Deploy the pipeline
+    Then Run the Pipeline in Runtime
+    Then Wait till pipeline is in running state
+    Then Open and capture logs
+    Then Verify the pipeline status is "Succeeded"
+    # TODO: this line needs to change. where to put the code?
```

Comment:
Is this `TODO` still relevant?

Reply:
any suggestion on where to include this step? I think it should be put in `common` package, but each package only allows one java class in this repo.